sdks/python/apache_beam/runners/common.py

```diff
@@ -661,8 +661,8 @@ def __init__(self, placeholder):
 
   def invoke_process(self,
                      windowed_value,  # type: WindowedValue
-                     restriction_tracker=None, # type: Optional[RestrictionTracker]
-                     watermark_estimator=None, # type: Optional[WatermarkEstimator]
+                     restriction=None,
+                     watermark_estimator_state=None,
```

Comment:
This is a tuple ?

Reply:
Its a user chosen value like the restriction. They can store anything they want to restore the state for the estimator. For system provided ones its typically the last reported timestamp.